test/metrics_test.py

```diff
@@ -119,6 +125,7 @@ def test_flush_with_attributes(self):
         send_metric.assert_called_once_with({
             'key': 'key',
             'attributes': {'what': 'test', 'k': 'v'},
+            'resources': {},
```

Comment:
Please add a `test_flush_with_resources` as well. Right now this only asserts that `{}` is passed through, but not that the actual value for resources gets there as there doesn't appear to be a test that verifies that.